charts/portworx/values.yaml

```diff
@@ -63,6 +72,7 @@ clusterToken:
   secretName: px-vol-encryption    # Name of kubernetes secret to be created. Requires clusterToken.create to be true.
   serviceAccountName: px-create-cluster-token # Service account name to use for post-install hook to create cluster token
 
-replicas: 3                         # Replica count for pvc-controller, stork, stork-scheduler.
-
 #requirePxEnabledTag: true               # if set to true, portworx will only install on nodes with px/enabled: true label. Not required in most scenarios.
+
+deleteStrategy:                           # Delete strategy for the portworx cluster
+  type:                                   # Valid values: Uninstall, UninstallAndWipe
```

Comment:
We need a default value here.

Reply:
Operator will decide default deleteStrategy if it's empty, the chart can leave it empty.

Reply:
Same question here. Can we delete it if it's taken care by the operator?

Reply:
we still need it so users can change it.